src/patterns/dosdp-patterns/cellCapableOfSensoryPerception.yaml

```diff
@@ -0,0 +1,42 @@
+pattern_name: cellCapableOfSensoryPerception
+pattern_iri: http://purl.obolibrary.org/obo/cl/cellCapableOfSensoryPerception
+
+description: A cell type that is capable of detecting a stimulus that is involved in sensory pereception (eg a receptor cell).
+example: CL_0000006 neuronal receptor cell
+
+contributors:
+  - https://orcid.org/0000-0001-5208-3432
+  - https://orcid.org/0000-0002-6601-2165
+  
+classes: 
+  native cell: "CL_0000003"
+  detection of stimulus involved in sensory perception: "GO:0050906"
+
+relations: 
+  capable of: "RO:0002215"
+
+vars:
+  cell-1: "'native cell'"
```

Comment:
this is a terrible pattern?

Do we really have a lot of instances of this??

Did this come from patternizer?